lib/krikri.rb

```diff
@@ -12,4 +12,5 @@ module Krikri
   autoload :OaiDcParser,    'krikri/parsers/oai_dc_parser'
   autoload :JsonParser,     'krikri/parsers/json_parser'
   autoload :ModsParser,     'krikri/parsers/mods_parser'
+  autoload :QdcParser,      'krikri/parsers/qdc_parser'
 end
```

Comment:
I am confused about autoloading.  I thought that this line from `lib/krikri/engine.rb` autoloads everything in `lib` and its subdirectories recursively, including the parsers:
`config.autoload_paths += Dir["#{config.root}/lib/**/"]`
Am I incorrect?